src/main/java/mike/cli/CommandLine.java

```diff
@@ -299,5 +344,25 @@ public static void help(String[] commandUsage) {
 
 				+ "  quit - exits the program\n");
 	}
+
+
+	public static boolean savePrompt (boolean prompt, Scanner cmdLine) {
+		while (prompt == true) {		
+			
+			String answer = cmdLine.nextLine();
+			
+			if (answer.equals("yes")) {
+				System.out.println("Proceeding.\n");
+				prompt = false;
+				break;
+			} else if (answer.equals("no")) {
+				System.out.println("Stopping.\n");
+				prompt = true;
+				break;
+			}
+			System.out.println("Invalid command. Type 'yes' to proceed, or 'no' to go back.");
+		}
```

Comment:
Does this compile without an else statement?

Reply:
It does. That final System.out.println("Invalid command...") basically acts as the else statement.